util/chunk/chunk.go

```diff
@@ -706,3 +706,22 @@ func (c *Chunk) ToString(ft []*types.FieldType) string {
 	}
 	return string(buf)
 }
+
+// AppendRows appends multiple rows to the chunk.
+func (c *Chunk) AppendRows(rows []Row) {
+	c.AppendPartialRows(0, rows)
+	c.numVirtualRows += len(rows)
+}
+
+// AppendPartialRows appends multiple rows to the chunk.
+func (c *Chunk) AppendPartialRows(colOff int, rows []Row) {
+	columns := c.columns[colOff:]
+	for index := 0; index < len(columns); index++ {
+		c.appendSel(colOff)
+	}
+	for i, dstCol := range columns {
```

Comment:
Why not `c.appendSel(colOff)` in this loop? It is of same effect to a stand alone loop like `for index := 0; index < len(columns); index++`.